### ISSUE TITLE
NullPointerException when deploying a BPMN, in ProcessMessageStartEventMessageNameValidator.java

### ISSUE THREAD

**Issue by nitram509** [camunda-cloud/zeebe#7789]
**Describe the bug**
When deploying the BPMN, Zeebe Broker crashes with NullPointerException.
Hint: the BPMN file was created, using Camunda Modeller v.4.8.1.8069

**To Reproduce**
* start Zeebe Broker and Monitor in e.g. Docker (via Docker Compose) 
* deploy test.bpmn via Monitor (see below)
* see stacktrace in logs

**Expected behavior**
* there should be no NPE, but rather a concrete error message, what's wrong with the BPMN
*
**Log/Stacktrace**

```STACKTRACE
zeebe_broker   | 2021-09-08 13:36:28.891 [Broker-0-StreamProcessor-1] [Broker-0-zb-actors-1] ERROR
zeebe_broker   |       io.camunda.zeebe.broker.process - Unexpected error while processing resource 'test.bpmn'
zeebe_broker   | java.lang.NullPointerException: null
zeebe_broker   | 	at io.camunda.zeebe.engine.processing.deployment.model.validation.ProcessMessageStartEventMessageNameValidator.validateMessageName(ProcessMessageStartEventMessageNameValidator.java:53) ~[zeebe-workflow-engine-1.0.3.jar:1.0.3]
zeebe_broker   | 	at io.camunda.zeebe.engine.processing.deployment.model.validation.ProcessMessageStartEventMessageNameValidator.lambda$validate$0(ProcessMessageStartEventMessageNameValidator.java:46) ~[zeebe-workflow-engine-1.0.3.jar:1.0.3]
zeebe_broker   | 	at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(Unknown Source) ~[?:?]
zeebe_broker   | 	at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source) ~[?:?]
zeebe_broker   | 	at java.util.stream.ReferencePipeline$2$1.accept(Unknown Source) ~[?:?]
zeebe_broker   | 	at java.util.ArrayList$Itr.forEachRemaining(Unknown Source) ~[?:?]
zeebe_broker   | 	at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Unknown Source) ~[?:?]
zeebe_broker   | 	at java.util.stream.AbstractPipeline.copyInto(Unknown Source) ~[?:?]
zeebe_broker   | 	at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source) ~[?:?]
zeebe_broker   | 	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(Unknown Source) ~[?:?]
zeebe_broker   | 	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(Unknown Source) ~[?:?]
zeebe_broker   | 	at java.util.stream.AbstractPipeline.evaluate(Unknown Source) ~[?:?]
zeebe_broker   | 	at java.util.stream.ReferencePipeline.forEach(Unknown Source) ~[?:?]
zeebe_broker   | 	at io.camunda.zeebe.engine.processing.deployment.model.validation.ProcessMessageStartEventMessageNameValidator.validate(ProcessMessageStartEventMessageNameValidator.java:46) ~[zeebe-workflow-engine-1.0.3.jar:1.0.3]
zeebe_broker   | 	at io.camunda.zeebe.engine.processing.deployment.model.validation.ProcessMessageStartEventMessageNameValidator.validate(ProcessMessageStartEventMessageNameValidator.java:25) ~[zeebe-workflow-engine-1.0.3.jar:1.0.3]
zeebe_broker   | 	at io.camunda.zeebe.model.bpmn.validation.ValidationVisitor.lambda$visit$0(ValidationVisitor.java:51) ~[zeebe-bpmn-model-1.0.3.jar:1.0.3]
zeebe_broker   | 	at java.util.ArrayList.forEach(Unknown Source) ~[?:?]
zeebe_broker   | 	at io.camunda.zeebe.model.bpmn.validation.ValidationVisitor.visit(ValidationVisitor.java:51) ~[zeebe-bpmn-model-1.0.3.jar:1.0.3]
zeebe_broker   | 	at io.camunda.zeebe.model.bpmn.traversal.TypeHierarchyVisitor.visit(TypeHierarchyVisitor.java:39) ~[zeebe-bpmn-model-1.0.3.jar:1.0.3]
zeebe_broker   | 	at io.camunda.zeebe.model.bpmn.traversal.ModelWalker.walk(ModelWalker.java:61) ~[zeebe-bpmn-model-1.0.3.jar:1.0.3]
zeebe_broker   | 	at io.camunda.zeebe.engine.processing.deployment.transform.BpmnValidator.validate(BpmnValidator.java:41) ~[zeebe-workflow-engine-1.0.3.jar:1.0.3]
zeebe_broker   | 	at io.camunda.zeebe.engine.processing.deployment.transform.DeploymentTransformer.transformResource(DeploymentTransformer.java:119) ~[zeebe-workflow-engine-1.0.3.jar:1.0.3]
zeebe_broker   | 	at io.camunda.zeebe.engine.processing.deployment.transform.DeploymentTransformer.transform(DeploymentTransformer.java:96) ~[zeebe-workflow-engine-1.0.3.jar:1.0.3]
zeebe_broker   | 	at io.camunda.zeebe.engine.processing.deployment.DeploymentCreateProcessor.processRecord(DeploymentCreateProcessor.java:100) ~[zeebe-workflow-engine-1.0.3.jar:1.0.3]
zeebe_broker   | 	at io.camunda.zeebe.engine.processing.streamprocessor.TypedRecordProcessor.processRecord(TypedRecordProcessor.java:54) ~[zeebe-workflow-engine-1.0.3.jar:1.0.3]
zeebe_broker   | 	at io.camunda.zeebe.engine.processing.streamprocessor.ProcessingStateMachine.lambda$processInTransaction$3(ProcessingStateMachine.java:296) ~[zeebe-workflow-engine-1.0.3.jar:1.0.3]
zeebe_broker   | 	at io.camunda.zeebe.db.impl.rocksdb.transaction.ZeebeTransaction.run(ZeebeTransaction.java:84) ~[zeebe-db-1.0.3.jar:1.0.3]
zeebe_broker   | 	at io.camunda.zeebe.engine.processing.streamprocessor.ProcessingStateMachine.processInTransaction(ProcessingStateMachine.java:286) ~[zeebe-workflow-engine-1.0.3.jar:1.0.3]
zeebe_broker   | 	at io.camunda.zeebe.engine.processing.streamprocessor.ProcessingStateMachine.processEvent(ProcessingStateMachine.java:254) ~[zeebe-workflow-engine-1.0.3.jar:1.0.3]
zeebe_broker   | 	at io.camunda.zeebe.engine.processing.streamprocessor.ProcessingStateMachine.tryToReadNextEvent(ProcessingStateMachine.java:219) ~[zeebe-workflow-engine-1.0.3.jar:1.0.3]
zeebe_broker   | 	at io.camunda.zeebe.engine.processing.streamprocessor.ProcessingStateMachine.readNextEvent(ProcessingStateMachine.java:210) ~[zeebe-workflow-engine-1.0.3.jar:1.0.3]
zeebe_broker   | 	at io.camunda.zeebe.util.sched.ActorJob.invoke(ActorJob.java:76) [zeebe-util-1.0.3.jar:1.0.3]
zeebe_broker   | 	at io.camunda.zeebe.util.sched.ActorJob.execute(ActorJob.java:39) [zeebe-util-1.0.3.jar:1.0.3]
zeebe_broker   | 	at io.camunda.zeebe.util.sched.ActorTask.execute(ActorTask.java:122) [zeebe-util-1.0.3.jar:1.0.3]
zeebe_broker   | 	at io.camunda.zeebe.util.sched.ActorThread.executeCurrentTask(ActorThread.java:94) [zeebe-util-1.0.3.jar:1.0.3]
zeebe_broker   | 	at io.camunda.zeebe.util.sched.ActorThread.doWork(ActorThread.java:78) [zeebe-util-1.0.3.jar:1.0.3]
zeebe_broker   | 	at io.camunda.zeebe.util.sched.ActorThread.run(ActorThread.java:191) [zeebe-util-1.0.3.jar:1.0.3]
zeebe_monitor  | 2021-09-08 13:36:28.960 ERROR 1 --- [nio-8082-exec-6] io.zeebe.monitor.rest.ExceptionHandler   : Command 'CREATE' rejected with code 'INVALID_ARGUMENT': Expected to deploy new resources, but encountered the following errors:
zeebe_monitor  | 'test.bpmn': null
```

TEST.BPMN file...

[test.bpmn.txt](https://github.com/camunda-cloud/zeebe/files/7129354/test.bpmn.txt)

**Environment:**
 - OS: Docker 3.6.0, OSX latest version
 - Zeebe Version: 1.0.4
 - Zeebe Simple Monitor: 2.1.0
 - Configuration: Docker compose setup from https://github.com/camunda-community-hub/zeebe-docker-compose/tree/master/simple-monitor , with version numbers adjusted, like described above

**Comment by npepinpe** [comment]
I believe this was just fixed? @remcowesterhoud / @saig0 is this the same thing?

**Comment by saig0** [comment]
This issue is a duplicate of #5510.

The bug is fixed by #7764 a few days ago :rocket:

**Comment by nitram509** [comment]
Awesome, thank you @saig0 @npepinpe for your quick response 👍
